src/modules/users/dto/users.dto.ts

```diff
@@ -57,3 +57,8 @@ export class UpdateAcceptedTosDto {
 	})
 	acceptedTosAt: string;
 }
+
+export class UpdateUserLangDto {
+	@IsIn(['nl', 'en'])
+	language: string;
```

Comment:
```suggestion
prefer to keep the syntax as close to existing fields as possible. I ttok this from the admin-core-api repo:
```
	@IsString()
	@ApiProperty({
		type: String,
		enum: LanguageCode,
	})
	language: LanguageCode;
```